src/iris/bin/sender.py

```diff
@@ -1504,7 +1504,7 @@ def main():
 
     maintain_workers(config)
 
-    gwatch_renewer_task = None
+    gwatch_renewer_task = config['sender'].get('disable_gwatch_renewer_task', False)
```

Comment:
Now that this takes values other than None, we should refactor line 1531 to `if not disable_gwatch_renewer and gwatch_renewer_task is not None`. Setting this to a boolean before initializing it to a task is somewhat confusing on first glance, especially since "True" results in no gwatch renewer task.